app/services/EnrolmentsStoreService.scala

```diff
@@ -45,7 +44,7 @@ class EnrolmentStoreServiceImpl @Inject()(connector: EnrolmentStoreConnector)(im
             case Nil => true
             case _ =>
               val expectedArrivalDate =
-                a.head.enrolmentTokenExpiryDate.get.minusDays(daysBetweenExpectedArrivalAndExpiry).toDateTime(DateTimeZone.UTC).getMillis
+                a.head.enrolmentTokenExpiryDate.get.minusDays(daysBetweenExpectedArrivalAndExpiry).atOffset(ZoneOffset.UTC).toLocalDateTime
```

Comment:
The code has changed slightly here. Now we use a LocalDateTime for currentDate. Before we used DateTime which has a date/time and offset. The enrolment token expiry date is also a LocalDateTime. Therefore, we should be able to compare currentDate and the expiry date directly.

Reply:
The question is, though, are the expiry dates all in GMT or a mixture of GMT and BST.

Reply:
I discussed this with Chris Poole and he said that all dates on the platform are UTC. As such it is fine to convert the expiry date to the UTC offset.